javascript/ql/test/library-tests/Portals/src/typeTracking/nonlocalDataflowRead.js

```diff
@@ -0,0 +1,12 @@
+// new portal exit node (i.e. caught with typetracking) annotated below
+
+var n = require('./index').NP;
+var np = new n();
+
+function getPortalReturn() {
+	return np.createNP();
+}
+
+function main() {
+  getPortalReturn().argFunc(); // here we have a nonlocal property access
+}							   // now, with typetracking, this is considered a portal exit node
```

Comment:
This comment makes sense as part of the PR, but not as part of checked-in code. I'd suggest omitting it (but keeping the one on the line above).